hazelcast/src/main/java/com/hazelcast/client/impl/management/ManagementCenterService.java

```diff
@@ -335,4 +337,63 @@ public void shutdownMember(Member member) {
                         MCGetTimedMemberStateCodec.decodeResponse(clientMessage).timedMemberStateJson)
         );
     }
+
+    /**
+     * Checks if local MC config (client B/W list) on a given member has the same ETag as provided.
+     *
+     * @param member    target member
+     * @param eTag      ETag value of MC config to match with (should be the latest value from MC)
+     * @return          operation future object with match result: <code>true</code> if config ETags match
+     */
+    @Nonnull
+    public CompletableFuture<Boolean> matchMCConfig(Member member, String eTag) {
+        checkNotNull(member);
+        checkNotNull(eTag);
+
+        ClientInvocation invocation = new ClientInvocation(
+                client,
+                MCMatchMCConfigCodec.encodeRequest(eTag),
+                null,
+                member.getAddress()
+        );
+        return new ClientDelegatingFuture<>(
+                invocation.invoke(),
+                serializationService,
+                clientMessage -> {
+                    MCMatchMCConfigCodec.ResponseParameters response =
+                            MCMatchMCConfigCodec.decodeResponse(clientMessage);
+                    return response.response;
+                },
+                true
```

Comment:
You should not need `deserializeResponse` as the response is not data.

Reply:
Thanks for finding this. Going to fix.

Reply:
Fixed.